app/2.0/start/quick-tour.md

```diff
@@ -38,7 +38,7 @@ an element name with a class, so you can add properties and methods to your cust
 element. The custom element's name **must start with an ASCII letter and
 contain a dash (-)**.
 
-<demo-tabs selected="0" name="qt-1-register" src="http://plnkr.co/edit/ScvcB4?p=preview">
+<demo-tabs selected="0" name="qt-1-register" src="http://plnkr.co/edit/Q4E8zO?p=preview">
```

Comment:
This plunker has plain text at the end of the script tag.
Also: might want to break the TODO comment lines in the plunker: they're really long and you need to scroll horizontally to read them.